src/components/common/@Icons/search.tsx

```diff
@@ -0,0 +1,14 @@
+const Search = () => {
+  return (
```

Comment:
다른 컴포넌트와 아이콘은 구별하기 쉽게 컴포넌트명 뒤에 `Icon`을 붙이는건 어떤가요?
```suggestion
const SearchIcon = () => {
  return (
```

Reply:
현재 생성되어있는 아이콘명 수정하겠슴니다!